install/CI/coverage

```diff
@@ -28,6 +28,11 @@ split_line(){
 
 # Start unit test.
 split_line "Start unit test"
+# api tests
+go test -v github.com/opensds/multi-cloud/api/pkg/utils/... -cover
```

Comment:
is it possible to run the go test simply for all sub-dirs and skip any error, if test file is not present

Reply:
That is what 'go test' does.